### DISCUSSION TITLE
Ingesting tool documentation for persistent RAG

### DISCUSSION THREAD

**Discussion by rjegge** [pieces-app/support#257] [General]
Apologies if I've missed this in my search of this discussion board and the docs....

With their knowledge cutoffs, etc, one of the things I'd most value (and pay for) is if I could use pieces to ingest documentation for new tools I adopt, or perhaps the latest version (eg, React 18, soon 19 / Nextjs 14, soon 15) as soon as docs are released. I'd like for knowledge of this documentation to persist in augmenting my own snippets and also whatever llm model(s) I'm using. Is this possible today, either through github in cases where docs are published in repo and/or when docs published on a website (eg, https://nextjs.org/docs)?

**Comment by mark-at-pieces** [comment]
Wonderful Idea, we have been chatting about this for a little bit now, but here is the linked issue internally, so that this thread can see when this is implemented and ready to go!(hmm looks like this discussion will not show the progress of the issue), however I will update this when we start the implementation and when it is ready.

https://github.com/pieces-app/os_server/issues/844
